R/simulate_catch.R

```diff
@@ -42,7 +43,7 @@ simulate_catch <- function(removals, fleet.props, dem_params, naa, options){
             remove <- removals*fleet.props[f]
 
             # Solve for F that removes catch
-            F_f <- findF_bisection(
+            F_f_tmp[,,f] <- findF_bisection(
```

Comment:
I would prefer outputting the results of `findF_bisection` to a temporary variable and then assigning to the temporary array at the end. Doesn't really matter at this level, but there is an I/O cost to reading data out of arrays.